documentation/2.0/content/rcuinfo.md

```diff
@@ -161,3 +182,205 @@ domainInfo:
         compInfoXMLLocation: wlsdeploy/rcu/config/MyComponentInfo.xml
         storageXMLLocation: wlsdeploy/rcu/config/MyStorage.xml
 ```
+
+### Advanced JRF database use cases
+
+In the following examples of the JRF data source sparse model, you can use it to further customize the JRF domain template data sources.
+
+#### Default template data source
+
+This is a sparse model for JRF data sources with the rcu prefix `FMW1`.  
+You will need to update at least the `URL`, `PasswordEncrypted`, and the `user` property value.  When you specify the value of `URL`, it 
+must be a valid `JDBC URL` format, which is different from the `rcu_db_conn_string` which can do not require the `jdbc:oracle:thin:...` part. 
```

Comment:
the `rcu_db_conn_string` which can do not require -> the `rcu_db_conn_string`, which does not require